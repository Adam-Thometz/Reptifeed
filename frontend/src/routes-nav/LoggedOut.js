import React from "react";
import { NavLink } from "react-router-dom";

const LoggedOut = () => {
  return (
    <div>
      <NavLink to='/login'>Login</NavLink>
      <NavLink to='/register'>Register</NavLink>
    </div>
  );
};

export default LoggedOut;