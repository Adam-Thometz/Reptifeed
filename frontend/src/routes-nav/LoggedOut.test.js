import React from "react";
import { render } from '@testing-library/react'
import { MemoryRouter } from 'react-router-dom';
import LoggedOut from "./LoggedOut";

test('it renders without crashing', () => {
  render(<MemoryRouter>
    <LoggedOut />
  </MemoryRouter>);
});

test('it matches snapshot', () => {
  const { asFragment } = render(
    <MemoryRouter>
      <LoggedOut />
    </MemoryRouter>
  );
  expect(asFragment()).toMatchSnapshot();
})