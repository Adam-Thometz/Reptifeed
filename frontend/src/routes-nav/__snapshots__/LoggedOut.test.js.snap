// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <a
      class=""
      href="/login"
    >
      Login
    </a>
    <a
      class=""
      href="/register"
    >
      Register
    </a>
  </div>
</DocumentFragment>
`;
